util/ubuntu-setup

```diff
@@ -156,6 +160,11 @@ fi
 # install all of the packages listed in required_packages.txt
 apt-get install $ARGS $(sed 's/#.*//;/^$/d' $BASE/util/ubuntu-packages.txt)
 
+if [ "$SYSTEM" = "ubuntu-14.04" ]; then
```

Comment:
Also a message telling people what's going on during this step would be nice (especially for debugging).
This is actually pretty important, this script is going to be rerun many times. We should make another conditional with something like this?
```
if command -v g++; then 
  echo "[WARN] You already have g++ symlinked. Not overwriting."
  exit 2
fi